vendor/vulkan/_gen/create_vulkan_odin_wrapper.py

```diff
@@ -85,6 +87,8 @@ def convert_type(t, prev_name, curr_name):
         "struct BaseInStructure":  "BaseInStructure",
         "struct wl_display": "wl_display",
         "struct wl_surface": "wl_surface",
+	"xcb_window_t": "u32",
+	"xcb_visualid_t": "u32",
```

Comment:
Should these be aliases or distinct types?

Reply:
XCB tends to use straight aliases a lot. Like, you can pass a xcb_window_t into anything that takes xcb_drawable_t. They're all just generic Ids